core/common/src/ConcurrentQuery.ts

```diff
@@ -298,7 +298,7 @@ export class BlobOptionsBuilder {
 }
 
 /** @internal */
-enum QueryParamType {
+export enum QueryParamType {
```

Comment:
@khanaffan is it good to start exporting this type? Looks like it may be intended internal only.
Simonas is using it just for asserting stuff further down in the code.